packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -313,6 +313,11 @@ export default class TabAdvanced extends Component {
 
   render () {
     const { container, isAdmin, vgpus, vm } = this.props
+    const virtualizationMode =
+      vm.virtualizationMode === 'hvm' && Boolean(vm.xenTools)
+        ? 'pvhvm'
+        : vm.virtualizationMode
```

Comment:
It would be interesting to abstract this logic, maybe in `utils/getVirtualizationModeLabel(vm)`.